signatures/sql.db

```diff
@@ -101,3 +101,6 @@ px_.*[[:space:]]*\(.*\$.*\)
 ovrimos_.*[[:space:]]*\(.*\$.*\)
 maxdb_.*[[:space:]]*\(.*\$.*\)
 db2_.*[[:space:]]*\(.*\$.*\)
+GRANT.*IDENTIFIED BY.*
```

Comment:
These rules appears to be an attempt at identifying credentials in SQL statements, which may be suitable for auditing applications, but I'd argue this would be in secrets rather than sql. Also, this file is generated by the makefile so you shouldn't add rules here.